.gitignore

```diff
@@ -191,6 +191,7 @@ node_modules/
 /ajax/libs/jquery-easing/1.3.0/
 /ajax/libs/jquery-easing/1.3.1/
 /ajax/libs/libphonenumber/libphonenumber*/
+/ajax/libs/slippry/v.1.2/
```

Comment:
@maruilian11 I think you can help rebase in this situation, I'll rebase it this time.